src/Microsoft.Azure.SignalR/Microsoft.Azure.SignalR.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.0;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
Shall we remove net 5.0 as well considering net 5.0 is also end of support already.

Reply:
Maybe also upgrade netsandard2.0 to netstandard2.1?

Reply:
netsandard2.0 supports net framework while netstandard2.1 does not

Reply:
Remove net 5.0? Not for now, Azure.Core also still supports net5.0 https://www.nuget.org/packages/Azure.Core/